Sprint-1/exercises/decimal.js

```diff
@@ -1,4 +1,17 @@
 const num = 56.5678;
+// Whole number part
+const wholeNumberPart = Math.floor(num); // 56
+
+// Decimal part
+const decimalPart = num - wholeNumberPart; // 0.5678
+
+// Rounded number
+const roundedNum = Math.round(num); // 57
+
+// Log the variables to the console
+console.log("Whole Number Part:", wholeNumberPart);
+console.log("Decimal Part:", decimalPart);
```

Comment:
This statement output: `Decimal Part: 0.5677999999999983`
This is normal because of *floating point arithmetic*.
Can you edit your code so that it output exactly 4 decimal places of `decimalPart`?

Reply:
Ok